index.js

```diff
@@ -106,6 +109,13 @@ module.exports = bundler => {
         };
 
         function copySingleFile(bundleDir, dest, filepath) {
+            if (path.basename(filepath) === 'manifest.json'){
```

Comment:
as opposed to having it check explicitly for a hardcoded filename, I would rather have it included in the config.  I would also rather replace with a regex instead of parsing to json and adding a node.

for example, the config could be something like:
```
{
  "outDirPattern": "**/dev",
  "staticPath: "config/development",
  "replace": {
    "files": ["manifest.json"],
    "field": "@@version",
    "value": "version"
  }
}
```

I wouldn't check the current manifest version against the package version, but just always perform a regex replace for @@version with pkg[value] whenever the current file was in the files list.